packages/react-vapor/src/components/editor/tests/CodeEditor.spec.tsx

```diff
@@ -119,8 +119,9 @@ describe('CodeEditor', () => {
             render(<CodeEditor id="anId" value="a value" mode={CodeMirrorModes.Python} />);
 
             expect(updateSpy).toHaveBeenCalledTimes(1);
-
+            
             userEvent.type(screen.getByRole('textbox'), 'new value');
+            fireEvent.blur(screen.getByRole('textbox'));

```

Comment:
```suggestion
            screen.getByRole('textbox').blur();
```

Would this work? Taken from [here](https://testing-library.com/docs/guide-events/#focusblur)

Reply:
Yes it may work indeed !  I do not think that will bring much value to the test as it is but I am willing to try it